.travis.yml

```diff
@@ -3,27 +3,15 @@ go:
   - 1.14.x
   - 1.15.x
   - tip
+arch:
+  - amd64
+  - ppc64le
 
 before_install:
   - go get -u github.com/vbatts/git-validation
 
-stages:
-  - validate
-  - build
-  - lint
-  - test
-
-jobs:
-  include:
-    - stage: build
-      script: make
-
-    - stage: lint
-      script: make lint BUILDTAGS=""
-
-    - stage: test
-      script: make test BUILDTAGS=""
-
-    - stage: validate
-      script: |
-        git-validation -run DCO,short-subject -v -range ${TRAVIS_COMMIT_RANGE}
+script:
+  - git-validation -run DCO,short-subject -v -range ${TRAVIS_COMMIT_RANGE}
+  - make
+  - make lint BUILDTAGS=""
+  - make test BUILDTAGS=""
```

Comment:
Won't this force the tests to disable SELinux?

Reply:
I know it was there from the beginning.  But it seems like we are testing only code compiled with the selinux disabled flag.
Perhaps we should run it twice. Once with BUILDTAGS="" and once without.

Reply:
Right, another dimension to the matrix, let's see if it works.

Reply:
fixed!